### DISCUSSION TITLE
where to store Android/Desktop common resources?

### DISCUSSION THREAD

**Discussion by guiv42** [helge17/tuxguitar#370] [General]
@helge17 I would appreciate your opinion about this.

I have recently reorganized all folders in this repo, to separate clearly what is specific to desktop app, specific to android app, and common (#166).
Then, I have implemented a mechanism to use translation files in common modules (#201). For this, I needed both Android and desktop apps to access the same translation files, and I added a symlink from Android to Desktop.
I need to do something similar for one dedicated resource file (tuning.xml), that needs to be shared and not duplicated (see #269).

Somehow, these symlinks from Android to Desktop seem to me inconsistent with the android/desktop/common organization of folders. Following approach would seem more logical to me, for shared resources:
- create a folder "common/resources"
- store in this folder the shared resources
  - folder "lang" containing translation files, currently in "desktop/tuxguitar/share/"
  - "tuning.xml" file
- in both android and desktop folders, symlink to the "common/resources" folder

?

**Comment by helge17** [comment]
I think your approach of moving shared resources to the `common` folder and linking to that folder from `desktop` and `android` is good!

**Reply by guiv42** [reply]
OK thanks, I'll do that

**Comment by guiv42** [comment]
It's done, for "lang" and "tunings" folders.
re-opening discussion: should the perl scripts in [misc](https://github.com/helge17/tuxguitar/tree/master/misc) folder also be updated? (I never use them...)

**Reply by helge17** [reply]
I will care about the build scripts.

**Reply by guiv42** [reply]
OK thanks

**Reply by helge17** [reply]
Done in commit a9fbd5271f55b135156613124781ec9b3f65afb3.
